.travis.yml

```diff
@@ -8,12 +8,12 @@ before_install:
   - conda update --yes conda
 
 install:
-  - conda env create -f ci/test-env.yml
+  - conda env create -f ci/conda_requirements.yml
   - source activate test-microsetta-public
-  - pip install "connexion[swagger-ui]"
-  - pip install ".[dev]"
```

Comment:
You need to retain this line, or move it into the `make install`

Reply:
`make install` performs `pip install -e .`

Reply:
Copied from the failed Travis build:
```
$ pip install -r ci/pip_requirements.txt
$ make install
$ make lint
flake8 microsetta_public_api
make: flake8: Command not found
Makefile:9: recipe for target 'lint' failed
```

This seems to suggest that the `"dev"` extras, which include `flake8`, are not installed.